src/modules/Kdump.rb

```diff
@@ -335,8 +335,17 @@ def calibrator
       end
     end
 
+    # Returns the Kdump memory limits
+    #
+    # It relies on the calibrator but it adjust the low memory limit when using fadump.
+    #
+    # @return [Hash] The hash contains the following keys: :min_low, :max_low,
+    #   :default_low, :min_high, :max_high, :default_high
     def memory_limits
-      calibrator.memory_limits
+      limits = calibrator.memory_limits
+      return limits unless using_fadump?
+
+      limits.merge(min_low: 0, max_low: total_memory)
```

Comment:
this can look a bit tricky, so maybe explaining comment would be nice? That we basically remove all limits with reason why ( jira entry and too complex kernel restrictions )?